loculus_values/environment_specific_values/demo.yaml

```diff
@@ -36,4 +36,6 @@ secrets:
 podPriorityClassName: staging
 seqSets:
   enabled: True
-  crossRef: null
\ No newline at end of file
+  crossRef: null
+ingest:
+  mirrorBucket: "https://hel1.your-objectstorage.com/loculus-public/mirror/"
```

Comment:
idk if we want to do this given we have been fine so far and that for pathoplexus a direct connection to NCBI seems good?

Reply:
I wasnt sure - so I left a direct connection to NCBI for staging and production but thought it might be good to use the mirror for all previews/demo just so that we dont get rate limited by NCBI in future

Reply:
Ah apologies - I didn't notice this was specific. So my original point is invalid. But actually it makes me think a bit: atm if e.g. NCBI blocked us we might eventually notice because main or PPX previews would be empty. If we did make this specific to staging/prod, might we not (because they will always have sequences anyway)? Or do we check argoCD well enough that we would?

IIUC think your change here doesn't actually effect Pathoplexus previews (which potentially defuses part of my point above, but I guess also means we could still get blocked by NCBI so 🤷 )

Reply:
No this is a good point - currently we dont have an alert if ingest fails anywhere so we also wouldn't know if production stops getting data. I had a quick chat with Cornelius and we think a good option might be to use the mirror everywhere and add a monitor to the mirror so we immediately know if that was not successful.

Reply:
If we do this then we'd want the mirroring to run significantly more regularly than currently IMO.

Reply:
What if we keep the current state as is - no mirroring and next week after setting up proper monitoring and more frequent mirroring we switch to the mirror?

Reply:
https://github.com/loculus-project/loculus/issues/3786